src/create-theme-provider.js

```diff
@@ -0,0 +1,92 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const isFunction = require('is-function');
```

Comment:
I'd go for not adding it. We really do not care about supporting any browsers that are this old. Babel doesn't care about it, Reaft doesn't care about it. So even if we out this in, chances are that things will fail long before this post even has an effect